### DISCUSSION TITLE
How can i remove app cmds and restore them

### DISCUSSION THREAD

**Discussion by lol219** [nextcord/nextcord#879] [Q&A]
Hi There,
Hope you are fine ,i have a question,i am wondering how to remove all slash commands using just code (or executing it via jishaku or any eval cmd) and restoring them without any reboot.
Can anybody help me. Thanks!

**Comment by TheKrafter** [comment]
This is what [Cogs](https://docs.nextcord.dev/en/stable/ext/commands/cogs.html) are for! 
The docs have everything you need to know, but for the sake of example:
```python
bot.unload_extension('cogs.example')
```
Would unload a cog named "example" (assuming `bot=commands.Bot()`, and 
```python
bot.load_extension('cogs.example')
```
would load it.

**Comment by lol219** [comment]
> This is what [Cogs](https://docs.nextcord.dev/en/stable/ext/commands/cogs.html) are for! 
> The docs have everything you need to know, but for the sake of example:
> ```python
> bot.unload_extension('cogs.example')
> ```
> Would unload a cog named "example" (assuming `bot=commands.Bot()`, and 
> ```python
> bot.load_extension('cogs.example')
> ```
> would load it.

You are right but they wont be gone
